src/parser/content-evaluator-module.ts

```diff
@@ -220,16 +217,18 @@ export class ContentEvaluatorModule implements Module {
     });
 
     const rawResponse = String(response.choices[0].message.content);
-    logger.info(`OpenAI raw response (using max_tokens: ${maxTokens}): ${rawResponse}`);
+    this.context.logger.info(`OpenAI raw response (using max_tokens: ${maxTokens}): ${rawResponse}`);
 
     const jsonResponse = JSON.parse(rawResponse);
 
     try {
       const relevances = Value.Decode(openAiRelevanceResponseSchema, jsonResponse);
-      logger.info(`Relevances by OpenAI: ${JSON.stringify(relevances)}`);
+      this.context.logger.info(`Relevances by OpenAI: ${JSON.stringify(relevances)}`);
       return relevances;
     } catch (e) {
-      logger.error(`Invalid response type received from openai while evaluating: ${jsonResponse} \n\nError: ${e}`);
```

Comment:
just wondering why we don't throw the logger if the SDK is handling the catch? Should we only use `throw new Error` I assume?

Reply:
No that would be possible most likely, I just didn't try it that way.